SetupBLT.cmake

```diff
@@ -94,12 +94,14 @@ include(${BLT_ROOT_DIR}/cmake/SetupCodeChecks.cmake)
 # bin directory for all executables and the location for fortran modules.
 
 # Set the path where all the headers will be stored
-set(HEADER_INCLUDES_DIRECTORY
-    ${PROJECT_BINARY_DIR}/include/
-    CACHE PATH
-    "Directory where all headers will go in the build tree"
-    )
-include_directories(${HEADER_INCLUDES_DIRECTORY})
+if ( ENABLE_COPY_HEADERS )
+    set(HEADER_INCLUDES_DIRECTORY
+        ${PROJECT_BINARY_DIR}/include/
+        CACHE PATH
+        "Directory where all headers will go in the build tree"
+        )
+    include_directories(${HEADER_INCLUDES_DIRECTORY})
+endif()
 
```

Comment:
i think this helps with #12 -- doesn't completely eliminate the issue, but makes it very unlikely to be encountered outside of basic testing.